boot/zephyr/Kconfig

```diff
@@ -422,7 +422,14 @@ config BOOT_KMU_KEYS_REVOCATION
 	help
 	  Enabling KMU key revocation backend.
 
-if !BOOT_SIGNATURE_USING_KMU
+config BOOT_SIGNATURE_USING_ITS
```

Comment:
Can we change that to NRF_, to avoid possible collision if something from upstream renders the same name?

Reply:
I think we shouldn't do that - note we do not do this with `BOOT_SIGNATURE_USING_KMU` which is an identical case, so we should follow the same pattern.

Reply:
Ok, changed after discussions with @nvlsianpu